synapse/storage/data_stores/main/events.py

```diff
@@ -1591,16 +1547,8 @@ async def locally_reject_invite(self, user_id: str, room_id: str) -> int:
         create a leave event for it.
         """
 
-        sql = (
-            "UPDATE local_invites SET stream_id = ?, locally_rejected = ? WHERE"
-            " room_id = ? AND invitee = ? AND locally_rejected is NULL"
-            " AND replaced_by is NULL"
-        )
-
         def f(txn, stream_ordering):
```

Comment:
(I know that `stream_ordering` is unused here, but I'm about to hook into it in another PR)